src/lib/expression/expression_utils.cpp

```diff
@@ -298,4 +299,24 @@ std::optional<AllTypeVariant> expression_get_value_or_parameter(const AbstractEx
   }
 }
 
+std::vector<std::shared_ptr<PQPSubqueryExpression>> collect_pqp_subquery_expressions(
```

Comment:
Changed the name:

```cpp
/**
 * @returns a vector of PQPSubqueryExpressions associated with @param expression.
 *          If @param expression is not of type PQPSubqueryExpression, its arguments will be searched for
 *          PQPSubqueryExpressions. Note, however, that it will not search for nested PQPSubqueryExpressions.
 */
std::vector<std::shared_ptr<PQPSubqueryExpression>> find_pqp_subquery_expressions(
    const std::shared_ptr<AbstractExpression>& expression);
```

Better?